src/PrestaShopBundle/Resources/views/Admin/Layout/default_layout.html.twig

```diff
@@ -113,7 +113,9 @@
              title="{{ maintenanceTitle|e('html') }}"
              href="{{ url('admin_maintenance') }}"
           >
-            <i class="material-icons">build</i>
+            <i class="material-icons"
+               style="{% if ps.frontOfficeAccessibleForAdmins|default(false) %}color: var(--green);{% endif %}"
```

Comment:
Hi 👋 In agreement with the designers, the link will be in orange (already handled by CSS), so we only need `<i class="material-icons">build</i>`. The green color and condition are no longer needed.

Reply:
@PoulainMaxime you should remove this extra style, the design team considered it useless and there is no more highlight related to `frontOfficeAccessibleForAdmins` anymore

However your fix about the variable `isMaintenanceEnabled` seems legit and should fix the initial bug. Can you please clean this style part and then the PR should be testable